hera_mc/cm_utils.py

```diff
@@ -14,7 +14,8 @@
 import os.path
 
 PAST_DATE = '2000-01-01'
-FUTURE_DATE = '2025-12-31'
+def _future_date():
+    return Time.now() + TimeDelta(100, format='jd')
```

Comment:
I did this because astropy.Time gives a warning for future dates over a certain UT1/UTC transition (?) or some reason.  So, I wanted to end the warnings that get printed out.  If I made a FUTURE_DATE close enough to not, it might not always be the FUTURE.